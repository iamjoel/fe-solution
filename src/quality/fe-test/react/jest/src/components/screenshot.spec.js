import React from 'react';
import renderer from 'react-test-renderer';
import Counter from "./screenshot";

// 第一次和多次后，和 DOM 的内容变化后，做更新。
it('should render correctly', () => {
  const tree = renderer
    .create(<Counter></Counter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
})