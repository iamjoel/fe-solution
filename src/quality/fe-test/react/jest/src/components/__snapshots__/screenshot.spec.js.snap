// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1
    className="title"
  >
    <span>
      ScreenShot Demo
    </span>
  </h1>
  <button
    className="minus-btn"
    onClick={[Function]}
  >
    -
  </button>
  <p
    className="value"
  >
    0
  </p>
  <button
    className="add-btn"
    onClick={[Function]}
  >
    +
  </button>
  <input
    className="input"
    onChange={[Function]}
    type="text"
    value={0}
  />
</div>
`;

exports[`should render correctly 1`] = `
<div>
  <h1
    className="title"
  >
    <span>
      ScreenShot Demo
    </span>
  </h1>
  <button
    className="minus-btn"
    onClick={[Function]}
  >
    -
  </button>
  <p
    className="value"
  >
    0
  </p>
  <button
    className="add-btn"
    onClick={[Function]}
  >
    +
  </button>
  <input
    className="input"
    onChange={[Function]}
    type="text"
    value={0}
  />
</div>
`;
